swap_meet/vendor.py

```diff
@@ -1,2 +1,142 @@
 class Vendor:
-    pass
\ No newline at end of file
+    """
+    A class to represent vendors participating in a swap meet. 
+
+    ...
+
+    Attributes
+    ----------
+    inventory: list
+        List of items that a vendor is trading. 
+
+    Methods
+    ----------
+    add(item):
+        Adds an item to inventory.
+    remove(item):
+        Removes an item from inventory.
+    get_by_category(category):
+        Makes a list of items that fall within the indicated category.
+    swap_items(vendor, my_priority, their_priority):
+        Swaps indicated items from inventory of two vendors.
+    swap_first_item(vendor):
+        Swaps the first item from inventory of two vendors. 
+    get_best_category(category):
+        Gets item in best condition from indicated category. 
+    swap_best_by_category(other, my_priority, their_priority):
+        Swaps the best item in a specified category from the invetory of two vendors.
+
+    """
+    
+    def __init__(self, inventory = None):
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self,item):
+        """Takes in an Item object, adds it to the inventory list, and returns the Item."""
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        """
+        Removes an item from inventory.
+
+            Parameter:
+                    item: An Item object
+            
+            Returns:
+                    False if Item is not in inventory. 
+                    Item if it is in inventory. 
+        """
+        if item not in self.inventory:
+            return False
+        else:
+            item_index = self.inventory.index(item)
+            return self.inventory.pop(item_index)
```

Comment:
Interesting approach grabbing the index position, then using pop to return the item!